packages/workbox-sw/index.mjs

```diff
@@ -20,8 +23,31 @@ class WorkboxSW {
   constructor(options) {
     this._options = Object.assign({
       debug: self.location.hostname === 'localhost',
-      pathPrefix: null,
+      modulePathPrefix: null,
+      modulePathCb: null,
+      disableModuleImports: false,
     }, options);
+    this._env = this._options.debug ? 'dev' : 'prod';
+
+    if (!this._options.disableModuleImports) {
+      this.loadModule('workbox-core');
+    }
+  }
+
+  /**
```

Comment:
How would you feel about using proxies here to do the loading? The code would be more concise without having to list out all the different possible submodules, and there's theoretically some benefit from being able to release new modules that could be loaded from the CDN without having to explicitly update `workbox-sw` to let it know about them.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Proxy

(Doesn't have to be in this PR—just something to think about.)